ember.js/2017-01/january-13.md

```diff
@@ -0,0 +1,135 @@
+# Ember Core Team 2017-01-13
+
+### Agenda
+
+- (public) FCP? https://github.com/emberjs/rfcs/pull/186 (SP)
+- (public) Update on [Router Service RFC](https://github.com/emberjs/rfcs/blob/master/text/0095-router-service.md) (and discussion of shipping in phases). (RJ)
+- (public) Standardizing microlib structure and build process (RM)
+- (public) Merge [JS Module API RFC](https://github.com/emberjs/rfcs/pull/176)? (TD)
+- (public) 2.12 deprecation checklist
+
+### Attendees
+
+MB: DG, GC, IT, RJ, TD, KS, SP, MM
+
+### Minutes
+
+#### FCP, Track unique history
+
+SP: I would like to go to FCP.
+
+KS: I would like the id to not be a counter, for example if a tab is reloaded you must be able to restore.
+
+SP: Ah you need a unique id
+
+KS: You could serialize to session storage and that would be a good enough scope. Session storage is not designed for this.
```

Comment:
"Session storage is not designed for this." => "Session storage is designed for this."